plugins/callback/unixy.py

```diff
@@ -92,12 +91,18 @@ def _process_result_output(self, result, msg):
     def v2_playbook_on_task_start(self, task, is_conditional):
         self._get_task_display_name(task)
         if self.task_display_name is not None:
-            self._display.display("%s..." % self.task_display_name)
+            if context.CLIARGS['check'] and self.get_option('check_mode_markers'):
```

Comment:
I think here you have to use `task.check_mode` since the behavior is configurable per task.
```suggestion
            if task.check_mode and self.get_option('check_mode_markers'):
```